docs/ReviewProcess.md

```diff
@@ -0,0 +1,187 @@
+if you are a first-time user of GitHub.
+
+### Prepare for review
+
+Before you post a pull request for review when you finish your work, make
+sure the following items are done:
+
+* The commits are formatted without coding standard issues. Although CI
+  formatting check is able to catch most issues, some still have to be
+  checked manually. Please see [Coding Standards](./CodingStandards.md) to
+  get familiar with LLPC coding standards and strictly follow them.
+
+* The commit descriptions are well written to describe the changes you
+  have made. Please see [Contributing to LLPC](./Contributing.md) for more
+  info. Reviewers would better understand your changes when those
+  descriptions are informative.
+
+* If your pull request is a large one to implement complex features or
+  code refactoring, a separate pull request adding the design document
+  is expected to be reviewed in advance. With this design document,
+  reviewers could know your roadmap and detailed design plans, improving
+  review efficiency. Don't forget to add design document links to commit
+  descriptions.
+
+* If your pull request is relevant to opened issues, please add issue
+  links to commit descriptions so reviewers know the background. Meanwhile,
+  this will facilitate administrators to track solutions and close issues
+  timely when they are resolved.
+
+### Assign reviewers
+
+After your preparation is done, post the pull request on GitHub. It is
+very likely others are more familiar with your change especially when
+they did relevant work in this functional area or have discussed your
+design plan with you when reviewing the design document. In such cases,
+you can suggest `Reviewers` in the right-top corner of the webpage of this
+pull request and we encourage you to do this.
+
+If you haven't got permission to assign reviewers then one of the active
+project participants (members, approvers or reviewers) will assign one for
+you. If you do have a suggestion for a reviewer then they can be requested
+in the comment field. If a reviewer is not assigned for 3 working days then
+you should ping the pull request.
+
+### Address review comments
+
+When your pull request is reviewed, you often receive review issues opened
+by reviewers. Please address them by updating your commits or continue to
+discuss them with reviewers if the suggestions are inappropriate. Once a
+review issue is either resolved or dropped, please click `Resolve conversation`
+to close it. If you forget to do so, it will cause misleading messages when
+reviewers recheck your updated commits.
+
+Don't make new commits to address review issues, just updating original
+commits and force-push them. This makes code history clear. It is helpful
+to write comments in a pull request with such wording `V#: Address review
+comments ..` when you update the pull request with new revisions. Reviewers
+therefore could easily track those revisions.

```

Comment:
In the interest of encouraging even more informative comments, how about:

> When you update the pull request with a new revision, it is helpful to also add a comment to the pull request mentioning what you did, for example (assuming the 4th revision to the pull request):
> ```
> v4: pulled out common code into ConvertFooToBar helper method
> ```

So still keep the intent of the recommendation here, but being slightly more explicit.

Reply:
Done.